Warehouse.Common/Extensions/StringExtension.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace Warehouse.Common.Extensions
+{
+    public static class StringExtension
```

Comment:
I think this class should be part of the application. These are business-specific methods.

Reply:
Done.